pages/data_life_cycle/preserving.md

```diff
@@ -54,7 +54,7 @@ When preparing data for preservation several requirements need to be fulfilled.
   * Include information about provenance.
   * Include sufficient licensing information.
   * Ensure that data is well organised.
-  * Ensure that a consistent naming scheme is used.
+  * Ensure that a consistent naming schema is used.
```

Comment:
Not sure about the others, but "naming scheme" is much more used than "naming schema" (1.540.000 Google hits vs. 55.100).

Reply:
"naming convention" is what is meant here, I think